### ISSUE TITLE
Of Nature's Wrath (shield)

### ISSUE THREAD

**Issue by maggotmvp** [LightsHope/server#782]
precursor This could just be extremely bad luck (highly unlikely imo)
I have been trying to find a shield of nature wrath (Of any level 1-60) for about 2~ months now on both factions multiple times per day. I've came to find that it does not seem to exist. I have however found a shield of almost all levels of shadow, fire, frost and even arcane??? wrath. I have also found that nature wrath rolls on different gear as well such as ring, head, boots, back, chest (anything with a "random bonus" roll) along with shadow fire frost arcane wrath.

In conclusion I'm fairly certain this is a bug considering the shields roll every other spell damage wrath. It seems very odd that it would ONLY be the shield which makes me think its possibly a bug.If you could shed any light on this that would be wonderful.

TY :D

p.s If you would like I can attach a long list of screen shots of every item slot that can roll nature wrath along with fire arcane frost including shield 

here is two examples   
https://imgur.com/hQf0zV4
https://imgur.com/LLw36zO

**Comment by GuybrushGit** [comment]
These shields can roll Nature's Wrath:
Dervish Buckler
Battleforge Shield
Scaled Shield
Lupine Buckler
Cutthroat's Buckler
Grizzly Buckler
Spiked Chain Shield

**Comment by maggotmvp** [comment]
ty

**Comment by GuybrushGit** [comment]
These shields cant roll it. Need a source if any of these are wrong.


id | name
-- | --
6559 | Bard's Buckler
6560 | Soldier's Shield
6571 | Scouting Buckler
6572 | Defender Shield
7108 | Infantry Shield
7330 | Infiltrator Buckler
7331 | Phalanx Shield
7463 | Sentinel Buckler
7465 | Knight's Crest
7495 | Captain's Buckler
7496 | Field Plate Shield
7536 | Champion's Wall Shield
7537 | Gothic Shield
9753 | Gypsy Buckler
9764 | Cadet Shield
9778 | Bandit Buckler
9790 | Raider's Shield
9804 | Superior Buckler
9816 | Fortified Shield
9843 | Banded Shield
9858 | Archer's Buckler
9873 | Renegade Shield
9899 | Jazeraint Shield
9918 | Brigade Defender
9935 | Embossed Plate Shield
9958 | Warmonger's Buckler
9974 | Overlord's Shield
10078 | Lord's Crest
10093 | Revenant Deflector
10158 | Mercurial Guard
10195 | Crusader's Shield
10204 | Heavy Lamellar Shield
10271 | Masterwork Shield
10362 | Ornate Shield
10363 | Engraved Wall
10364 | Templar Shield
10365 | Emerald Shield
10366 | Demon Guard
10367 | Hyperion Shield
13254 | Astral Guard
14902 | Saltstone Shield
14912 | Brutish Shield
14916 | Jade Deflector
14930 | Lofty Shield
14947 | Warbringer's Shield
14954 | Bloodforged Shield
14964 | High Chief's Shield
14973 | Glorious Shield
14982 | Exalted Shield
15113 | Rigid Buckler
15123 | Robust Buckler
15145 | Ghostwalker Buckler
15307 | Feral Buckler
15332 | Wrangler's Buckler
15342 | Pathfinder Guard
15352 | Headhunter's Buckler
15367 | Trickster's Protector
15486 | War Torn Shield
15494 | Bloodspattered Shield
15504 | Outrunner's Shield
15512 | Grunt's Shield
15530 | Sentry's Shield
15543 | Wicked Chain Shield
15552 | Thick Scale Shield
15563 | Pillager's Shield
15569 | Marauder's Crest
15584 | Sparkleshell Shield
15592 | Steadfast Buckler
15604 | Ancient Defender
15618 | Bonelink Wall Shield
15621 | Gryphon Mail Buckler
15633 | Formidable Crest
15648 | Ironhide Shield
15657 | Merciless Shield
15667 | Impenetrable Wall
15675 | Magnificent Guard
15687 | Triumphant Shield
15887 | Heroic Guard

**Comment by maggotmvp** [comment]
tyty unlucky

**Comment by maggotmvp** [comment]
tyty just unlucky
